src/inputs/NumberField.tsx

```diff
@@ -140,7 +143,7 @@ export function NumberField(props: NumberFieldProps) {
     value: valueRef.current.wip ? valueRef.current.value : value === undefined ? Number.NaN : value / factor,
     // // This is called on blur with the final/committed value.
     onChange: (value) => {
-      onChange(formatValue(value, factor, numFractionDigits, numIntegerDigits));
+      onChange(formatValue(value, factor, numFractionDigits, numIntegerDigits, positiveOnly));
```

Comment:
Thought/suggestion: I think there is another place we call formatValue. Should we do this in both places? I'd be curious on the user experience if we do both vs just what you have.

Reply:
I dont see any difference on behavior, but still added the change. Thanks